src/central-server/admin-service/application/src/main/resources/permissions.yml

```diff
@@ -54,22 +54,22 @@
 - APPROVE_OWNER_CHANGE_REQUEST: [ XROAD_REGISTRATION_OFFICER ]
 - REVOKE_OWNER_CHANGE_REQUEST: [ XROAD_REGISTRATION_OFFICER ]
 ##
-- VIEW_MEMBER_CLASSES: [ XROAD_REGISTRATION_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
-- ADD_MEMBER_CLASS: [ XROAD_SYSTEM_ADMINISTRATOR ]
-- EDIT_MEMBER_CLASS: [ XROAD_SYSTEM_ADMINISTRATOR ]
-- DELETE_MEMBER_CLASS: [ XROAD_SYSTEM_ADMINISTRATOR ]
+- VIEW_MEMBER_CLASSES: [ XROAD_SECURITY_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
+- ADD_MEMBER_CLASS: [ XROAD_SECURITY_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
+- EDIT_MEMBER_CLASS: [ XROAD_SECURITY_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
+- DELETE_MEMBER_CLASS: [ XROAD_SECURITY_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
 ##
-- VIEW_CONFIGURATION: [ XROAD_REGISTRATION_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
-- VIEW_CONFIGURATION_MANAGEMENT: [ XROAD_SYSTEM_ADMINISTRATOR, XROAD_SECURITY_OFFICER, XROAD_REGISTRATION_OFFICER ]
+- VIEW_CONFIGURATION: [ XROAD_SECURITY_OFFICER, XROAD_SYSTEM_ADMINISTRATOR ]
+- VIEW_CONFIGURATION_MANAGEMENT: [ XROAD_SYSTEM_ADMINISTRATOR, XROAD_SECURITY_OFFICER ]
 - VIEW_TRUSTED_ANCHORS: [ XROAD_SYSTEM_ADMINISTRATOR, XROAD_SECURITY_OFFICER ]
 - UPLOAD_TRUSTED_ANCHOR: [ XROAD_SECURITY_OFFICER ]
 - DELETE_TRUSTED_ANCHOR: [ XROAD_SECURITY_OFFICER ]
 - DOWNLOAD_TRUSTED_ANCHOR: [ XROAD_SYSTEM_ADMINISTRATOR, XROAD_SECURITY_OFFICER ]
-- VIEW_INTERNAL_CONFIGURATION_SOURCE: [ XROAD_SYSTEM_ADMINISTRATOR, XROAD_SECURITY_OFFICER, XROAD_REGISTRATION_OFFICER ]
+- VIEW_INTERNAL_CONFIGURATION_SOURCE: [ XROAD_SYSTEM_ADMINISTRATOR, XROAD_SECURITY_OFFICER ]
 - VIEW_EXTERNAL_CONFIGURATION_SOURCE: [ XROAD_SYSTEM_ADMINISTRATOR, XROAD_SECURITY_OFFICER ]
 - GENERATE_SOURCE_ANCHOR: [ XROAD_SECURITY_OFFICER ]
 - DOWNLOAD_SOURCE_ANCHOR: [ XROAD_SECURITY_OFFICER ]
-- UPLOAD_CONFIGURATION_PART: [ XROAD_SECURITY_OFFICER, XROAD_REGISTRATION_OFFICER ]
+- UPLOAD_CONFIGURATION_PART: [ XROAD_SECURITY_OFFICER ]
```

Comment:
Seems like this line is missing `XROAD_SYSTEM_ADMINISTRATOR` role.
According to [spec](https://github.com/nordic-institute/X-Road/blob/release-7.4/doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md#55-managing-the-contents-of-a-configuration-part) upload & download should both have `XROAD_SYSTEM_ADMINISTRATOR` & `XROAD_SECURITY_OFFICER` roles